monai/networks/utils.py

```diff
@@ -615,6 +626,103 @@ def convert_to_torchscript(
     return script_module
 
 
+def convert_to_trt(
+    model: nn.Module,
+    precision: str,
+    inputs_shape: Sequence[Sequence[int]],
```

Comment:
I think this `inputs_shape` arg can be None when converting to `TorchScript` with `torch.jit.script()`, right?
If True, please add None support for this arg in this function and the `export` functions, and unit test for `None`.

Thanks.

Reply:
Actually, the `inputs_shape` for `convert_to_trt` function cannot be `None`, since it is needed to generate the placeholder for the torch_tensorrt compiler.  But for the `trt_export` function, since we will parse the shape from the parser, it could be None, and the type hint is already supported `None` input. As for the `_export` function, it doesn't have this parameter explictly as input.

Reply:
OK. Thanks.